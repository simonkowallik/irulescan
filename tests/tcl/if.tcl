if {1} {
    expr 1
} elseif {2} {
    expr 2
} else {
    expr 3
}

# test .. then .. support

if {10} then {
    expr 10
}
elseif {20} then {
    expr 20
}
else {
    expr 30
}

if {100} {
    expr 100
} elseif {200} {
    expr 200
}
elseif {300} {
    expr 300
}
else {
    expr 400
}

if {1000}
{
    expr 1000
}
elseif {2000}
{
    expr 2000
}
else
{
    expr 3000
}
