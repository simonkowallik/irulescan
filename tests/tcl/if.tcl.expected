WARNING: Unquoted expr at `1` in `expr 1`
WARNING: Unquoted expr at `2` in `expr 2`
WARNING: Unquoted expr at `3` in `expr 3`
WARNING: Unquoted expr at `10` in `expr 10`
WARNING: Unquoted expr at `20` in `expr 20`
WARNING: Unquoted expr at `30` in `expr 30`
WARNING: Unquoted expr at `100` in `expr 100`
WARNING: Unquoted expr at `200` in `expr 200`
WARNING: Unquoted expr at `300` in `expr 300`
WARNING: Unquoted expr at `400` in `expr 400`
WARNING: badly formed command at `else` in `else`

